package.json

```diff
@@ -20,13 +20,17 @@
     "node": "4.4.x",
     "npm": "3.9.x"
   },
-  "browserslist": ["last 2 versions", "ie 11"],
+  "browserslist": [
+    "last 2 versions",
+    "ie 11"
+  ],
   "config": {
     "port": 4200,
     "externalplugins": ""
   },
   "dependencies": {
     "babel-polyfill": "6.23.0",
+    "baconjs": "0.7.95",
```

Comment:
Not sure about the latest numbers but I'd use Rx or Kefir instead of Beacon due to performance issues.  https://github.com/Reactive-Extensions/RxJS/blob/master/doc/mapping/bacon.js/whyrx.md